src/components/WithdrawSBtc.tsx

```diff
@@ -13,18 +13,19 @@ import {
 import { useConnect } from '../lib/auth';
 import { NETWORK, mocknet } from '../lib/constants';
 import { getSbtcWalletAddress } from '../lib/sbtc';
+import { P2TROut } from '@scure/btc-signer/lib/payment';
 
-export function WithdrawSBtc({ assetContract }) {
-  const spinner = useRef();
-  const amountSatsRef = useRef();
+export function WithdrawSBtc({ assetContract }: { assetContract: string }) {
+  const spinner = useRef<HTMLDivElement>(null);
+  const amountSatsRef = useRef<HTMLInputElement>(null);
   const [loading, setLoading] = useState(false);
-  const [status, setStatus] = useState();
+  const [status, setStatus] = useState<string>();
   const { userSession } = useConnect();
   const [signature, setSignature] = useState('');
 
-  const signMessage = async e => {
+  const signMessage = async (e: React.UIEvent) => {
```

Comment:
better just `e: UIEvent`
import the parts from React

Reply:
it's better to use it this way since UIEvent is a native type in DOM global scope but React.UIEvent is a synthetic event so I thought it's more readable to mention that this is the synthetic event

Reply:
OK